CMakeLists.txt

```diff
@@ -1593,8 +1609,10 @@ set(INSTALLED_EXAMPLES
     ${CMAKE_CURRENT_SOURCE_DIR}/examples/sctp/sctp-client.c
     ${CMAKE_CURRENT_SOURCE_DIR}/examples/sctp/sctp-server-dtls.c
     ${CMAKE_CURRENT_SOURCE_DIR}/examples/echoclient/echoclient.c
-    ${CMAKE_CURRENT_SOURCE_DIR}/examples/server/server.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/test_apps/server/test_server.c
```

Comment:
Please group these... test_ next to each other. Also add a `# comment` for why examples/client and server are not in `add_executable` above (because they are already covered here). And confirm this with building or @haydenroche5